docs/concepts/html/classes.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Classes
+description: A handy macro to handle classes
+---
+
+## Classes
+
+The struct `Classes` can be used to deal with HTML classes.
+
+When pushing a string to the set, `Classes` ensures that there is one element
+for every class even if a single string might contain multiple classes.
+
+`Classes` can also be merged by using `Extend` (i.e.
+`classes1.extend(classes2)`) or `push()` (i.e. `classes1.push(classes2)`). In
+fact, anything that implements `Into<Classes>` can be used to push new classes
+to the set. This includes: `&'static str`, `String` and `Cow<'static, str>`,
+but also: `Vec<impl Into<Classes>>`, `&[impl Into<Classes>]`,
+`Option<impl Classes>` and `&Option<impl Classes>`. For example
```

Comment:
```suggestion
`Option<impl Into<Classes>>` and `&Option<impl Into<Classes>>`. For example
```

I'm worried that this might overwhelm the user (it's a lot of "and"s in a row). What if we just use the `DOCUSAURUS_CODE_TABS` below to demonstrate the different built-in implementations?

I feel like the tab for "Interpolated" is pretty misleading anyway because it's literally just a string...

Reply:
Fixed in 42da556

> I'm worried that this might overwhelm the user (it's a lot of "and"s in a row). What if we just use the DOCUSAURUS_CODE_TABS below to demonstrate the different built-in implementations?

I removed the first "and". I believe it is still grammatically correct if not better but I'm not a native English speaker.

After re-reading I think it is fine as it is because:
1. the first example with the code tabs is already showing an example with the Option
2. the last example within a component shows the option but also another Classes

> I feel like the tab for "Interpolated" is pretty misleading anyway because it's literally just a string...

Yes... I think it's best to remove it entirely. 7da9ea7

Reply:
(I'm totally open to suggestion :grin: !)

Reply:
:thinking: maybe we could merge both examples into one. Use only examples withing component because `classes!` taken alone is actually pretty difficult to understand imo.

And we make every example (single, multiple, literals, options, classes...)

Reply:
> thinking maybe we could merge both examples into one.

That's what I meant, yeah. That way we can explain both at the same time and it looks a bit prettier.